docs/eessi.md

```diff
@@ -0,0 +1,72 @@
+# EESSI Guide
+
+## How to Load EESSI
+EESSI can be initialised using the following method.
+
+The EESSI environment is sourced (in a non-reversible way) by running:
+```[bash]
+source /cvmfs/software.eessi.io/versions/2023.06/init/bash
+```
+
+This is non-reversible because it:
+- Changes your `$PATH`, `$MODULEPATH`, `$LD_LIBRARY_PATH`, and other critical environment variables.
+- Sets EESSI-specific variables such as `EESSI_ROOT`.
+
+This is the recommended method because it:
+- Detects your CPU architecture and OS.
+- Detects and configures GPU support.
+- Prepares the full EESSI software stack.
+- Sets up Lmod (environment module system).
+
+The [EESSI docs](https://www.eessi.io/docs/using_eessi/setting_up_environment/) offer another method to load EESSI, in addition to one above. The alternative method only initialises the Lmod module system and does not load a platform-specific setup. For these reasons, it is recommened to use the method detailed above.
+
+Successful environment setup will show `{EESSI 2023.06}` at the start of your CLI.
+
+To deactivate your EESSI environment you can either restart your shell using `exec bash` or exit the shell by `exit`.
+
+## GPU Support with EESSI
+To enable GPU support, you need a site-specific build that has CUDA enabled. For a guide on how to do this, please refer to [docs/image-build.md](../image-build.md).
+
+### Using GPUs
+All CUDA-enabled software in EESSI expects CUDA drivers in a specific `host_injections` directory.
+
+#### To expose the NVIDIA GPU drivers:
+Use the `link_nvidia_host_libraries.sh` script, provided by EESSI, to symlink your GPU drivers into `host_injections`.
+```[bash]
+/cvmfs/software.eessi.io/versions/2023.06/scripts/gpu_support/nvidia/link_nvidia_host_libraries.sh
+```
+Rerun this script when your NVIDIA GPU drivers are updated. It is also safe to rerun at any time as the script will detect if the driver versions have already been symlinked.
```

Comment:
So, this is an actual meaty question: does EACH user need to run this, i.e. is it a shell-specific thing? Or is it actually a "run-once" thing? Does it survive reboots (not rebuilds!)?

If its a run-once thing which survives reboots, we should probably do it ourslves, like we do for CVMFS config in:
- ansible/roles/eessi/tasks/configure.yml
- ansible/roles/compute_init/files/compute-init.yml
(search for EESSI)

Reply:
Actually if the answer is "we should do it ourselves" I don't think that blocks merging this PR actually. But I think we should understand the answer under the current ticket, and create a follow-on ticket if necessary.